attributes/default.rb

```diff
@@ -17,11 +17,11 @@
 
 default['grafana']['manage_install'] = true
 default['grafana']['install_type'] = 'file' # file | package | source
-default['grafana']['version'] = '2.1.2'
```

Comment:
please change in a yet another commit, same with urls and checksums

Reply:
done